Makefile

```diff
@@ -39,6 +39,7 @@ bender_targs += -t rtl
 bender_targs += -t test
 bender_targs += -t mchan
 bender_targs += -t cluster_standalone
+bender_targs += -t pulp
```

Comment:
Is the target serving the iCache itself? Would it make sense to use a more specific target (something like `icache_pulp`) in that case?

Reply:
I can also just add this module by default without requiring the extra target.

Reply:
Has this changed? If we still need the tag probably we have to port it also in the system we integrate the cluster into (it is fine, we just have to know it :) )

Reply:
This is no longer necessary, I removed it now.